inspirehep/base/searchext/mappings/hep.json

```diff
@@ -78,28 +95,79 @@
                         "properties": {
                             "value": {
                                 "type": "string",
-                                "index": "not_analyzed"
+                                "index": "not_analyzed",
+                                "copy_to": ["doi"]
                             }
                     }
                 },
+                "doi": {
+                    "type": "string",
+                    "index": "not_analyzed"
+                },
                 "report_numbers": {
                     "type": "object",
                     "properties": {
                         "value": {
                             "type": "string",
-                            "index": "not_analyzed"
+                            "index": "not_analyzed",
+                            "copy_to": ["reportnumber"]
                         }
                     }
                 },
+                "reportnumber": {
+                    "type": "string",
+                    "index": "not_analyzed"
+                },
                 "arxiv_eprints": {
                     "type": "object",
                     "properties": {
                         "value": {
                             "type": "string",
-                            "index": "not_analyzed"
+                            "index": "not_analyzed",
+                            "copy_to": ["arXiv"]
+                        },
+                        "categories": {
+                            "type": "string",
+                            "index": "not_analyzed",
+                            "copy_to": ["subject"]
+                        }
+                    }
+                },
+                "arXiv": {
+                    "type": "string"
+                },
+                "isbns": {
+                    "type": "object",
+                    "properties": {
+                        "value": {
+                            "type": "string",
+                            "copy_to": ["isbn"]
```

Comment:
Same thing. Would be great to index both the original string, then with and without dashes.